django-backend/fecfiler/scha_transactions/migrations/0008_schatransaction_parent_transaction.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.12 on 2022-06-23 13:59
```

Comment:
I think you're going to need to delete this migration and your 0009 migration.  your changes will need to be subsequent migrations.

Reply:
:+1: